docs/style-guide.adoc

```diff
@@ -195,6 +195,22 @@ RETURN "Cypher"
 RETURN 'Cypher'
 ----
 
+. Write the value null in lower case.
```

Comment:
In its final form, this sentence is a bit hard to read... can null be highlighted somehow?